samples/github/workflow.py

```diff
@@ -0,0 +1,76 @@
+"""Trigger GitHub Action workflows, and receive workflow events."""
+
+from autokitteh.github import github_client
+
+
+def start_github_action(event):
+    """Start a GitHub action workflow.
+
+    See the following link for more information:
+    https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+
+    This function is preconfigured as the entry-point for an AutoKitteh trigger,
+    but it's currently commented out in the "autokitteh.yaml" file. To activate it,
+    uncomment the following lines:
+
+        # - name: github_push
+        #   connection: github_conn
+        #   event_type: push
+        #   call: workflow.py:start_github_action
+
+    Additionally, it requires a named workflow YAML file to be present in a relevant GitHub repository.
+
+    Example workflow YAML file (in the repo's ".github/workflows" directory):
+
+        on: workflow_dispatch
+        jobs:
+          job-name:
+            runs-on: ubuntu-latest
+            steps:
+              - run: echo "Do stuff"
+
+    Args:
+        event: GitHub event data (e.g. new pull request or push event).
+    """
+    repo = event.data.repo.full_name
+    # TODO: ENG-1631
+    ref = event.data.repo.default_branch  # Branch name or tag
+    workflow_file = "dispatch.yml"  # .github/workflows/dispatch.yml
+
+    g = github_client("github_conn")
+    repo = g.get_repo(repo)
+    workflow = repo.get_workflow(workflow_file)
+
+    print(f"Triggering workflow: {workflow_file}")
```

Comment:
Applicable in `on_github_workflow_dispatch` too, but not the other 2 functions.

```suggestion
    print("Triggering workflow:", workflow_file)
```

Reply:
Done